PF/Assignment1/ConcordanceMain.java

```diff
@@ -0,0 +1,23 @@
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.Set;
+
+//main class
+public class ConcordanceMain {
+	// main function
+	public static void main(String args[]) {
+		Scanner scan = new Scanner(System.in);
+		String input = new String();
+		System.out.println("Enter the string");
+		input = scan.next();
+		int length = input.length();
```

Comment:
ACTION: There is no need to create a new variable to store input length. You can directly pass it as  new ConcordanceOfCharacters(input, input.length());